osdu/.htaccess

```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://github.com/Accenture/OSDU-Ontology [R=302,L]
```

Comment:
Note that this is pointing at your github page, not the plain TTL file.  So if URLs like https://w3id.org/osdu#xxx are dereferenced, they would just point at the github UI.  Is that ok?

Reply:
I'll fix it.